pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1165,7 +1157,7 @@ func addWebhookPwds(buildObject *api.KieAppBuildObject) {
 func getProcessMigrationTemplate(cr *api.KieApp, serversConfig []api.ServerTemplate) (processMigrationTemplate *api.ProcessMigrationTemplate, err error) {
 	if deployProcessMigration(cr) {
 		processMigrationTemplate = &api.ProcessMigrationTemplate{}
-		processMigrationTemplate.ImageURL = constants.RhpamPrefix + "-process-migration" + constants.RhelVersion + ":" + cr.Status.Applied.Version
+		processMigrationTemplate.ImageURL = constants.ProcessMigrationDefaultImageURL + ":" + cr.Status.Applied.Version
```

Comment:
```suggestion
		processMigrationTemplate.ImageURL = constants.ProcessMigrationDefaultImage + ":" + cr.Status.Applied.Version
```

Reply:
no, for imageURL we want the full url with registry... so we use `ProcessMigrationDefaultImageURL` here. Honestly, I should probably remove `ProcessMigrationDefaultImage` from constants as its only used in test file.

Reply:
@jakubschwan I've removed `ProcessMigrationDefaultImage` from constants to avoid confusion. thanks!